### ISSUE TITLE
[FR] Keeping old mods visible on the list so it's easier to see what is still missing.

### ISSUE THREAD

**Issue by JohnEdwa** [Assistant/ModAssistant#265]
After an update, ModAssistant shows me the old mods I had, but only if there already has been an update. It would be helpful if it also showed those that haven't been updated yet so I can see what is still missing.

You know, something like this:
![dfafdfsdsdfsdf](https://user-images.githubusercontent.com/5417183/97293103-e0a28980-1854-11eb-8f2a-342ebabaec0b.png)

Or have them all under a category such as "Unsupported", "Not Updated" or "For 1.11.0".

**Comment by pravic** [comment]
+1 for this. Otherwise it's really hard to recall what mods were installed before the update.

**Comment by plsMajor** [comment]
> Otherwise it's really hard to recall what mods were installed before the update.

![image](https://user-images.githubusercontent.com/32457983/128244770-33217fee-693d-48bb-93d1-e2b9e64a38ad.png)
Isn't that what these are for?

**Comment by JohnEdwa** [comment]
It doesn't fix the issue that a mod that isn't updated yet just disappears from the list completely, so you can't know what mods you are missing without having to memorize them all and try to see if they are on the list yet. Like right now, YUR Fit and MultiplayerExtensions aren't, but I know that only because I remember those and I manually went through the mod list.

So when the next major update hits, it would really handy if there was just:

* ~~MultiplayerExtensions~~
* ~~Chroma~~
* ~~MappingExtensions~~
* ~~NoodleExtensions~~

at the bottom of the mod list so I can open the manager, check they are still unupdated and not bother as half the songs I like don't work and the rest I can't play in multiplayer.

**Comment by pravic** [comment]
@plsMajor No, those checkboxes work as long as the mods are installed AND they are available on the website. Once BS is updated, there are no mods available for the new version and the mods disappear from the list.

That is the point of this ticket: "Save selected mods" should remember the previously selected mods even if they are not available currently on the website. If we need to remove some mod, we will wait until it appears in the list and then we will remove it using ModAssistant.
